torchvision/csrc/cuda/DeformConv_cuda.cu

```diff
@@ -698,7 +698,7 @@ static void compute_grad_offset(
   }
 }
 
-static std::tuple<at::Tensor, at::Tensor> deform_conv_backward_input_cuda(
+static std::tuple<at::Tensor, at::Tensor> deform_conv2d_backward_input_cuda(
```

Comment:
@fmassa This is a safe renaming but FYI.

Reply:
This renaming LGTM